BenGWeeks/Zapp.ie
# Task 96100: Move "Send Zap" custom action to Zapp.ie repo (and separate pipeline)

### Description

Moved code used in Alice for Accounts into Zapp.ie. The Azure function is used to send zaps to users within Logic Apps using a custom connector.

### Screenhots
The Code and Functions
![image](https://github.com/user-attachments/assets/49a134fa-7991-447f-9b13-8a82da98ea70)
![image](https://github.com/user-attachments/assets/be831307-0abd-4387-add9-e5616439490c)


### Related workitems

- [Task 96100](https://dev.azure.com/EvrosPowerPlatformTeam/Default/_workitems/edit/96100): Move "Send Zap" custom action to Zapp.ie repo (and separate pipeline)

Fixes # (issue)

### Type of Change


### Test Instructions

To test this PR:

Ensure you have Azure-core-tools installed to run Azure Functions locally

1. Open VS code
2. navigate to the functions folder (cd functions)
3. type 'npm install'
4. Type 'func start' you will see the following in the terminal
![image](https://github.com/user-attachments/assets/e1d78f27-7cf1-4145-990a-547dce3b99f4)

5. Open Postman to test the function local

### Checklist:
